src/server/game/Entities/Player/Player.cpp

```diff
@@ -26930,6 +26930,18 @@ uint8 Player::GetMostPointsTalentTree() const
     return maxIndex;
 }
 
+bool Player::IsHealerTalentSpec() const
+{
+    uint8 tree = GetMostPointsTalentTree();
+    return ((getClass() == CLASS_DRUID && tree == 2) || (getClass() == CLASS_PALADIN && tree == 0) || (getClass() == CLASS_PRIEST && tree <= 1) || (getClass() == CLASS_SHAMAN && tree == 2));
+}
+
+bool Player::IsTankTalentSpec() const
+{
+    uint8 tree = GetMostPointsTalentTree();
+    return ((getClass() == CLASS_PALADIN && tree == 1) || (getClass() == CLASS_WARRIOR && tree == 2) || (getClass() == CLASS_DRUID && tree == 1) || (getClass() == CLASS_DEATH_KNIGHT && tree == 1));
+}
+
```

Comment:
What functionality do these have for the AFK reports?

Reply:
I did not add these two functions O_O
Errors come from here https://github.com/azerothcore/azerothcore-wotlk/commit/bcd5c567d07b313e1c672b0371ee02f24f250be4